seed/challenges/intermediate-bonfires.json

```diff
@@ -61,7 +61,7 @@
         "diff([1, 2, 3, 5], [1, 2, 3, 4, 5]);"
       ],
       "tests": [
-        "expect(diff([1, 2, 3, 5], [1, 2, 3, 4, 5])).to.be.a('array');",
+        "assert(typeof(diff([1, 2, 3, 5], [1, 2, 3, 4, 5])) === \"object\", 'The result should be an array.');",
```

Comment:
This could use [`assert.isArray`](http://chaijs.com/api/assert/#isArray) instead.